lib/internal/repl/completion.js

```diff
@@ -731,35 +731,15 @@ function includesProxiesOrGetters(expr, exprStr, evalFn, ctx, callback) {
 
     if (astProp.type === 'Literal') {
       // We have something like `obj['foo'].x` where `x` is the literal
-
-      if (safeIsProxyAccess(obj, astProp.value)) {
-        return cb(true);
-      }
-
-      const propDescriptor = ObjectGetOwnPropertyDescriptor(
-        obj,
-        `${astProp.value}`,
-      );
-      const propHasGetter = typeof propDescriptor?.get === 'function';
-      return cb(propHasGetter);
+      return propHasGetterOrIsProxy(obj, `${astProp.value}`, cb);
```

Comment:
Should this not just be

```suggestion
      return propHasGetterOrIsProxy(obj, astProp.value, cb);
```

Reply:
I've added it just in case, it is probably not necessary :+1: 
(PS: does it hurt to have it in though? :thinking:)